app/jobs/regular/chat_channel_delete.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Jobs
+  class ChatChannelDelete < ::Jobs::Base
+    def execute(args = {})
+      chat_channel = ::ChatChannel.with_deleted.find_by(id: args[:chat_channel_id])
+
+      # this should not really happen, but better to do this than throw an error
+      if chat_channel.blank?
+        Rails.logger.warn("Chat channel #{args[:chat_channel_id]} could not be found, aborting delete job.")
+        return
+      end
+
+      DistributedMutex.synchronize(
+        "delete_chat_channel_#{chat_channel.id}",
+        validity: 20.minutes
+      ) do
+        Rails.logger.debug("Deleting webhooks and events for channel #{chat_channel.id}")
```

Comment:
```suggestion
        Rails.logger.debug { "Deleting webhooks and events for channel #{chat_channel.id}" }
```

Using a block here will save a string allocation because the block is only called when the log level is met.

Reply:
I feel like this is kind of a micro-optimization? We don't do that at all in core